cli/changelog/0.68.0.md

```diff
@@ -0,0 +1,3 @@
+### Features
+
+- Support for the @cors directive.
```

Comment:
I think this commit would be new in fixes: https://github.com/grafbase/grafbase/commit/0cda754021ce9b02afe25ef836dd76f7d291b7cc